osu.Game/Screens/Select/PlaySongSelect.cs

```diff
@@ -108,5 +128,13 @@ protected override bool OnStart()
 
             return true;
         }
+
+        private Mod getActiveAutoMod()
+        {
+            if (Mods.Value.Any(m => m.GetType() == Ruleset.Value.CreateInstance().GetAutoplayMod()?.GetType())) return Ruleset.Value.CreateInstance().GetAutoplayMod();
+            else if (Mods.Value.Any(m => m.GetType() == Ruleset.Value.CreateInstance().GetCinemaMod()?.GetType())) return Ruleset.Value.CreateInstance().GetCinemaMod();
```

Comment:
I don't agree with this being handled locally as done here. The higher-level logic of mod selection/deselection should be deselecting one when the other is selected, via `IncompatibleMods` or similar. (see #7155) Otherwise we're going to end up with local fixes every time an issue like this arises.

That was also another problem the previous attempt at this had.